api/control/v1alpha1/types.go

```diff
@@ -110,6 +110,14 @@ type BindingPolicySpec struct {
 	// expectation is not met.
 	// +optional
 	WantSingletonReportedState bool `json:"wantSingletonReportedState,omitempty"`
+
+	// MaxWrappedSize is the maximum size of the object containing the wrapped objects
+	// that are to be downsynced. By default the size is the same as that of
+	// ocm's manifestwork:
+	// https://github.com/open-cluster-management-io/ocm/blob/ff77abd31208aaf328b1dc0826b66caf9cdf6dcd/pkg/work/webhook/common/validator.go#L16
+	// +optional
+	// +kubebuilder:default=512000
+	MaxWrappedSize int64 `json:"maxWrappedSize,omitempty"`
```

Comment:
as suggested in our slack chat, I think this should be environment configuration and set as argument of the transport controller rather than set by users who create BPs.

Reply:
Done